service/src/main/kotlin/fi/espoo/evaka/daycare/controllers/DaycareController.kt

```diff
@@ -72,16 +73,24 @@ class DaycareController(
         return db.connect { dbc -> dbc.read { it.getUnitFeatures() } }
     }
 
-    @PutMapping("/{daycareId}/features")
-    fun putFeatures(
+    @PostMapping("/unit-features")
+    fun postUnitFeatures(
         db: Database,
         user: AuthenticatedUser,
-        @PathVariable daycareId: DaycareId,
-        @RequestBody features: Set<PilotFeature>
+        @RequestBody request: UpdateFeaturesRequest
     ) {
-        Audit.UnitFeaturesUpdate.log(targetId = daycareId)
-        accessControl.requirePermissionFor(user, Action.Unit.UPDATE_FEATURES, daycareId)
-        db.connect { dbc -> dbc.transaction { it.setUnitFeatures(daycareId, features) } }
+        Audit.UnitFeaturesEnable.log(targetId = request.unitIds)
+        accessControl.requirePermissionFor(user, Action.Unit.ENABLE_FEATURES, request.unitIds)
```

Comment:
This should probably use `Action.Unit.DISABLE_FEATURES` in some cases

Reply:
I've reverted the naming changes to logging and to the action. The renaming was originally done for separating the enabling and disabling to separate routes, but I didn't end up using that solution; as such, the renaming is no longer relevant.